dashboard/lib/script_constants.rb

```diff
@@ -17,6 +17,16 @@ module ScriptConstants
   CSP17_UNIT2_NAME = 'csp2-2017'.freeze
   CSP17_UNIT3_NAME = 'csp3-2017'.freeze
 
+  # CSF family names
+  COURSEA = 'coursea'.freeze
```

Comment:
Correct! To be slightly more specific we `include ScriptConstants` into the Unit class

Reply:
Including a module into a class makes all of the module's members accessible on the class.

Reply:
yes, for modules `require` and `include` are different. this page describes what happens when you include a module: https://ruby-doc.org/core-2.6.3/Module.html

you can also `require` a module, in which case it behaves a lot like requiring a class, e.g.: https://github.com/code-dot-org/code-dot-org/blob/52c18b0b364f56455c066d11568e4a7dc1af8db2/aws/ci_build#L36

there are many blog posts on the internet about this, the main thing I would suggest is that you search for "ruby" and not "rails" because this is actually plan ruby stuff not specific to rails.